src/SDKs/Storage/Storage.Tests/Tests/StorageAccountTests.cs

```diff
@@ -1288,5 +1288,121 @@ public void StorageAccountCreateWithHttpsOnlyTest()
+                Assert.NotNull(account.Encryption);
+                Assert.True(account.Encryption.Services.Blob.Enabled);
+                Assert.True(account.Encryption.Services.File.Enabled);
+                Assert.Equal("Microsoft.Keyvault", account.Encryption.KeySource);
+
+                // Disable Encryption.
+                updateParameters = new StorageAccountUpdateParameters
+                {
+                    Encryption = new Encryption
+                    {
+                        Services = new EncryptionServices { Blob = new EncryptionService { Enabled = true }, File = new EncryptionService { Enabled = true } },
+                        KeySource = "Microsoft.Storage"
+                    }
+                };
+                account = storageMgmtClient.StorageAccounts.Update(rgname, accountName, updateParameters);
+                StorageManagementTestUtilities.VerifyAccountProperties(account, false);
+                Assert.NotNull(account.Encryption);
+                Assert.True(account.Encryption.Services.Blob.Enabled);
+                Assert.True(account.Encryption.Services.File.Enabled);
+                Assert.Equal("Microsoft.Storage", account.Encryption.KeySource);
+
+
+                updateParameters = new StorageAccountUpdateParameters
+                {
+                    Encryption = new Encryption
+                    {
+                        Services = new EncryptionServices { Blob = new EncryptionService { Enabled = false }, File = new EncryptionService { Enabled = false } },
+                    }
+                };
+                account = storageMgmtClient.StorageAccounts.Update(rgname, accountName, updateParameters);
+                StorageManagementTestUtilities.VerifyAccountProperties(account, false);
+                Assert.Null(account.Encryption);
+
+                storageMgmtClient.StorageAccounts.Delete(rgname, accountName);
+            }
+        }
+        [Fact]
+        public void StorageAccountOperationsTest()
+        {
+            var handler = new RecordedDelegatingHandler { StatusCodeToReturn = HttpStatusCode.OK };
+
+            using (MockContext context = MockContext.Start(this.GetType().FullName))
+            {
+                var resourcesClient = StorageManagementTestUtilities.GetResourceManagementClient(context, handler);
+                var storageMgmtClient = StorageManagementTestUtilities.GetStorageManagementClient(context, handler);
+                var keyVaultMgmtClient = StorageManagementTestUtilities.GetKeyVaultManagementClient(context, handler);
+
+                // Create storage account with hot
+                string accountName = TestUtilities.GenerateName("sto");
+                var rgname = StorageManagementTestUtilities.CreateResourceGroup(resourcesClient);
+
+                var ops = storageMgmtClient.Operations.List();
+
+                Assert.NotNull(ops);
+                Assert.NotNull(ops.GetEnumerator());
```

Comment:
Operations.List() will always return the full operations that storage RP supports. I added extra check for two of common operations.